ui.apps/src/main/content/jcr_root/apps/etoolbox-query-kit/components/console/queryStatistic/queryStatistic.html

```diff
@@ -0,0 +1,95 @@
+<sly data-sly-use.queryStatisticModel="com.exadel.etoolbox.querykit.core.models.query.QueryStatisticModel"></sly>
```

Comment:
(spelling) Please rename to "...statistics" everywhere

Reply:
All changed